docs/test-doubles.md

```diff
@@ -25,7 +27,31 @@ Allows you to override the output of a callable.
 ```bash [Example]
 function test_example() {
   function code() {
-      ps a | grep bash
+    ps a | grep bash
+  }
+
+  mock ps<<EOF
+PID TTY          TIME CMD
+13525 pts/7    00:00:01 bash
+24162 pts/7    00:00:00 ps
+EOF
+
+  assert_equals "13525 pts/7    00:00:01 bash" "$(code)"
+}
+```
+:::
+
+## unmock
```

Comment:
In my honest opinion, I wouldn't document the unmock as it could cause confusion among our users. The option is there, but the unmock should be automatic. Documenting it might lead our users to think they need to perform this process manually.

Reply:
Ok, agreed. I will simply remove it from the docs - and it will be like an easter egg feature. You can use it in your tests, but you dont have to, as it will happen automatically after each test anyway.